forms-flow-review/src/components/TaskList/TasklistTable.tsx

```diff
@@ -92,6 +67,58 @@ const TaskListTable = () => {
   const isTaskListLoading = useSelector((state: any) => state.task.isTaskListLoading);
 
   const taskvariables = selectedFilter?.variables ?? [];
+  const getCellValue = (column: Column, task: Task) => {
+  const { sortKey } = column;
```

Comment:
@Bonymol-aot please check the sonar issues, can we reduce the cognitive complexity down to 15